rkt/fetch.go

```diff
@@ -34,7 +34,10 @@ var (
 	cmdFetch = &cobra.Command{
 		Use:   "fetch IMAGE_URL...",
 		Short: "Fetch image(s) and store them in the local store",
-		Run:   runWrapper(runFetch),
+		Long: `Locates and downloads remote ACIs and their attached signatures
+over HTTPS. If the ACI image is available locally, the image will not
```

Comment:
It is not necessary over HTTPS. It can be a local file: `rkt fetch /tmp/myfile.aci`. I suggest:

> Locates and fetches ACIs and their attached signatures. The ACIs can be located via meta-discovery, at a specific URL, at a local file, or via Docker.

Also, `s/locally/in the local store/`. So it is not confused with a local file.

The text comes from: https://github.com/coreos/rkt/blob/master/Documentation/subcommands/fetch.md
So it needs to be updated too.